### PR TITLE
Feature/1.26.x

### PR DESCRIPTION
**Pull request by deregges**

### CODE REVIEW

File: src/main/java/database/repositories/VersionRepository.java [hunk 1]
```diff
@@ -15,4 +16,7 @@
      * @return a database stored {@link VersionInformation} with the name of versionName
      */
     VersionInformation findVersionInformationByVersionName(String versionName);
+
+    @Query("match (n:VersionInformation) where not (n)<-[:FOLLOWED]-() return n")
```
Comment (ramzi277) [inline src/main/java/database/repositories/VersionRepository.java]: Please adjust this query to also include the child nodes

Comment (ramzi277) [reply src/main/java/database/repositories/VersionRepository.java]: I would suggest tu just query the nodes versionname here and then rely on findVersionInformationByVersionName(String versionName) again

File: src/main/java/core/DependencyExtractor.java [hunk 2]
```diff
@@ -59,10 +60,10 @@ public VersionInformation analyseClasses(List<String> classFiles, String current
      * @return the corresponding CTClass
      * @throws IOException then the class file on the given is not found
      */
-    private CtClass getOrCreateCtClass(String classFile) throws IOException {
-        CtClass ctClass = classPool.getOrNull(classFile);
+    private CtClass getOrCreateCtClass(Path classFile) throws IOException {
+        CtClass ctClass = classPool.getOrNull(classFile.toString());
```
Comment (deregges) [inline src/main/java/core/DependencyExtractor.java]: Does this even work? Class file name for class query (jdoc states that query string should be class name).
Kept the pattern nonetheless

Comment (ramzi277) [reply src/main/java/core/DependencyExtractor.java]: Removed this method entirely since the classpool already provides this.

**Pull request merged**